packages/bits/spec/components/menu/menu.e2e.ts

```diff
@@ -202,14 +107,12 @@ describe("USERCONTROL Menu", () => {
                 });
 
                 it("should select and close menu when selecting menu action item", async () => {
-                    await menu.toggleMenu();
```

Comment:
have you checked that menu is opened in this case, it is possible that those tests where  "await menu.toggleMenu();" was deleted become false positive

Reply:
Discussed offline - "await menu.toggleMenu();" was moved to _beforeEach_